generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json

```diff
@@ -175,6 +175,24 @@
     {
       "path": "../examples/assembly/compile.tsconfig.json"
     },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
+    {
+      "path": "../examples/assembly/compile.tsconfig.json"
+    },
```

Comment:
these changes are the result of https://github.com/eclipse/che/issues/15916
it's very annoying bug...

Reply:
OK then -> https://github.com/eclipse/che-theia/pull/655